src/api/goods.ts

```diff
@@ -10,10 +10,20 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => {
+      const newGoods = [...goods];
+
+      newGoods.sort((a, b) => {
+        return a.name.localeCompare(b.name);
+      });
+
+      newGoods.length = 5;
+
+      return newGoods;
+    });
```

Comment:
this code could be optimized by using `.slice()` method. to change the length of array is not a good idea. also there is no need to create a copy of goods before sorting, you can work directly with `goods`